src/ShellCommand/ShellCommandRunner.php

```diff
@@ -314,21 +314,13 @@ private function _uploadToS3($localFilePath, $targetUrl)
     $bucket = $urlParts['host'];
     $path   = preg_replace('/^\//', '', $urlParts['path']);
 
-    // get the s3 region.
-    $customData = $this->shellCommand->getCustomData();
-    if (isset($customData['s3Region'])) {
-      $s3Region = $customData['s3Region'];
-    } else {
-      throw new Exception("An s3Region is required. Please add this to the command's customData.");
-    }
-
     // Upload!
     $options = [
       'credentials' => [
         'key'    => $this->s3Key,
         'secret' => $this->s3SecretKey,
       ],
-      'region'  => $s3Region,
+      'region'  => 'us-east-1', // hardcoded since Tourbuzz only uses this regions for S3 atm.
```

Comment:
i think the right answer actually is to expose it as an option the same way that s3Key and s3SecretKey are used, and then make the hard-code in the client code that bootstraps the ShellCommandRunner.

However I don't really understand the use cases for region swapping, so maybe it's possible that the region would need to change on a per-ShellCommand basis? I am not sure, but I don't think that too many people juggle regions a ton, nor do too many people use ShellCommand. So I think the above approach (moving it into an option and letting client hard-code it) is a great fix for now that safely de-couples this from tourbuzz.